S21/osamaibraheem11/s21.py

```diff
@@ -0,0 +1,26 @@
+import csv
+
+vowels = ["A","E","I","O","U","Á","É","Í","Ó","Ú"]
```

Comment:
Could we rename this as "irishVowels"?

Reply:
Yeah, maybe "irishVowels" is more clear.